Crash when/after invoking Google Maps navigation

<!-- Fill in the following form by adding your text below the explanation comments. -->
<!-- You can use the preview tab above to review your issue before submitting it. -->

## Bug description 
<!-- Enter a summarized description of what the bug/problem is, that you found -->
Reported by a beta user on support mail.


## Reproduce 
### Steps to reproduce the problem
<!-- Describe step-by-step how to reproduce the problem -->
- Open cache details of a cache
- Select Navigation -> Navigation (Driving)


### Actual result after these steps 
<!-- Describe the actual issue/problem/behavior in detail, which happens after the steps above -->
Bounces back to cache details page and often crashes directly after

### Expected result after these steps 
<!-- Describe what you expected to happen instead (correct behavior) -->
Navigation app should be started

## c:geo version 
<!-- You will find the c:geo version in c:geo Menu -> About c:geo -->
2021.05.05-RC

## Reproducible 
<!-- Yes / No (or describe under what conditions) -->
Yes, for the affected user, but not for me.

## System information 
<!-- Attach system information here if available (see c:geo Menu -> About c:geo -> Swipe right to System) -->
<!-- Keep the apostrophe at beginning and end to have it properly formatted -->


```
--- System information ---
c:geo version: 2021.05.05-RC

Device:
-------
- Device type: Pixel 4 XL (coral, google)
- Available processors: 8
- Android version: 11
- Android build: RQ2A.210405.005
- Sailfish OS detected: false
- Google Play services: disabled - 21.15.15 (150400-371058782)
- HW acceleration: enabled (default state)

Sensor and location:
-------
- Low power mode: inactive
- Compass capabilities: yes
- Rotation vector sensor: present
- Orientation sensor: present
- Magnetometer & Accelerometer sensor: present
- Direction sensor used: orientation

Program settings:
-------
- Hide caches: own/found disabled archived
- Hide waypoints: -
- Set language: en_US (system default)
- System date format: M/d/yy
- Debug mode active: no
- Live map mode: true
- OSM multi-threading: true / threads: 4
- Global filter: display all caches
- Last backup: Apr 29, 7:16 PM
- Routing mode: Walk
-Settings: v5, Count:154
- Map: CyclOSM
  - Id: cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider$CyclosmMapSource
  - Atts: CyclOSM, map data OpenStreetMap contributors
  - Theme: none

Services:
-------
- Geocaching sites enabled:
   geocaching.com: Logged in (Login OK) / PREMIUM
   Geocaching.com Adventure Lab
- Geocaching.com date format: MM/dd/yyyy
- Routing: external / BRouter installed: false
- Installed c:geo plugins:  none

Permissions & paths:
-------
- Fine location permission: granted
- Write external storage permission: granted
- System internal c:geo dir: /data/user/0/cgeo.geocaching (50.4 GB free) v2 internal isDir(9 entries)
- Legacy User storage c:geo dir: /storage/emulated/0/cgeo (50.4 GB free) v2 external non-removable isDir(7 entries)
- Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (50.4 GB free) v2 external non-removable isDir(908 entries)
- Internal theme sync (is turned off): /data/user/0/cgeo.geocaching/MapThemeData (50.4 GB free) v2 internal isDir(0 entries)
- Public Folders: #11
  - BASE: /cgeo (User-Defined)[/cgeo[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo, Av:true, files:>=5, dirs:>=9, totalFileSize:>=814.5 MB, free space: 50.4 GB, files on device: 13511677)
  - OFFLINE_MAPS: /cgeo/maps (Default)[/cgeo/maps[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/maps]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fmaps, Av:true, files:2, dirs:1, totalFileSize:773.5 MB, free space: 50.4 GB, files on device: 13511677)
  - OFFLINE_MAP_THEMES: /cgeo/maps/_themes (Default)[/cgeo/maps/_themes[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/maps/_themes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fmaps%2F_themes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - LOGFILES: /cgeo/logfiles (Default)[/cgeo/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/logfiles]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Flogfiles, Av:true, files:1, dirs:0, totalFileSize:44.7 KB, free space: 50.4 GB, files on device: 13511677)
  - GPX: /cgeo/gpx (Default)[/cgeo/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/gpx]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fgpx, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - BACKUP: /cgeo/backup (Default)[/cgeo/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/backup]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fbackup, Av:true, files:2, dirs:1, totalFileSize:41.0 MB, free space: 50.4 GB, files on device: 13511677)
  - FIELD_NOTES: /cgeo/field-notes (Default)[/cgeo/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/field-notes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Ffield-notes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - SPOILER_IMAGES: /cgeo/GeocachePhotos (Default)[/cgeo/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/GeocachePhotos]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2FGeocachePhotos, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - ROUTING_BASE: /cgeo/routing (Default)[/cgeo/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/routing]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Frouting, Av:true, files:0, dirs:1, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - ROUTING_TILES: /cgeo/routing/segments4 (Default)[/cgeo/routing/segments4[PERSISTABLE_FOLDER(ROUTING_BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/routing/segments4]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Frouting%2Fsegments4, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: 13511677)
  - TEST_FOLDER: [Legacy]/data/user/0/cgeo.geocaching/files/unittest (Default)[/data/user/0/cgeo.geocaching/files/unittest[FILE#1:p-file:///data/user/0/cgeo.geocaching/files::/unittest]] (Uri: file:///data/user/0/cgeo.geocaching/files/unittest, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 50.4 GB, files on device: -1)
- Map render theme path:
- PersistedDocumentUris: #1
- TRACK: null
- Persisted Uri Permissions: #1
  - content://com.android.externalstorage.documents/tree/primary%3Acgeo (Mar 19, 1:06 PM):RW
- Database: /data/user/0/cgeo.geocaching/databases/data (v94, Size:41.3 MB) on system internal storage
--- End of system information ---
```

## Additional context 
<!-- (optional, remove if not applicable) log files, reference to other similar issues, projects, sources, etc. -->
Log file contains:
```
--------- beginning of crash
05-06 13:30:10.923 22816 22816 E AndroidRuntime: FATAL EXCEPTION: main
05-06 13:30:10.923 22816 22816 E AndroidRuntime: Process: cgeo.geocaching, PID: 22816
05-06 13:30:10.923 22816 22816 E AndroidRuntime: android.view.InflateException: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Error inflating class <unknown>
05-06 13:30:10.923 22816 22816 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Error inflating class <unknown>
05-06 13:30:10.923 22816 22816 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.createView(LayoutInflater.java:852)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.createView(LayoutInflater.java:774)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:928)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:948)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1002)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:657)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at cgeo.geocaching.apps.navi.GoogleNavigationApp$GoogleNavigationDrivingApp$1.getView(GoogleNavigationApp.java:95)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.AbsListView.obtainView(AbsListView.java:2387)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.ListView.measureHeightOfChildren(ListView.java:1417)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.ListView.onMeasure(ListView.java:1323)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.widget.AlertDialogLayout.tryOnMeasure(AlertDialogLayout.java:146)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.widget.AlertDialogLayout.onMeasure(AlertDialogLayout.java:71)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:747)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:3402)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:2219)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2504)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1948)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8177)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:796)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:731)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:938)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040048 a=-1}
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:997)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.content.res.TypedArray.getDrawable(TypedArray.java:981)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.View.<init>(View.java:5535)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.view.ViewGroup.<init>(ViewGroup.java:697)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:255)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:251)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:247)
05-06 13:30:10.923 22816 22816 E AndroidRuntime: 	... 48 more
05-06 13:30:26.010  7152  7152 E AndroidRuntime: FATAL EXCEPTION: main
05-06 13:30:26.010  7152  7152 E AndroidRuntime: Process: cgeo.geocaching, PID: 7152
05-06 13:30:26.010  7152  7152 E AndroidRuntime: android.view.InflateException: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Error inflating class <unknown>
05-06 13:30:26.010  7152  7152 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #2 in cgeo.geocaching:layout/cacheslist_item_select: Error inflating class <unknown>
05-06 13:30:26.010  7152  7152 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.createView(LayoutInflater.java:852)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.createView(LayoutInflater.java:774)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:928)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:948)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1002)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:657)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at cgeo.geocaching.apps.navi.GoogleNavigationApp$GoogleNavigationDrivingApp$1.getView(GoogleNavigationApp.java:95)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.AbsListView.obtainView(AbsListView.java:2387)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.ListView.measureHeightOfChildren(ListView.java:1417)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.ListView.onMeasure(ListView.java:1323)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.widget.AlertDialogLayout.tryOnMeasure(AlertDialogLayout.java:146)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.widget.AlertDialogLayout.onMeasure(AlertDialogLayout.java:71)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:747)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.measure(View.java:25466)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:3402)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:2219)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2504)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1948)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8177)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:796)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:731)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:938)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040048 a=-1}
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:997)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.content.res.TypedArray.getDrawable(TypedArray.java:981)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.View.<init>(View.java:5535)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.view.ViewGroup.<init>(ViewGroup.java:697)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:255)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:251)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	at android.widget.RelativeLayout.<init>(RelativeLayout.java:247)
05-06 13:30:26.010  7152  7152 E AndroidRuntime: 	... 48 more
--------- beginning of main
```

--- a/main/res/layout/cacheslist_item_select.xml
+++ b/main/res/layout/cacheslist_item_select.xml
@@ -5,7 +5,6 @@
     android:id="@+id/one_cache"
     android:layout_width="fill_parent"
     android:layout_height="wrap_content"
-    android:background="?background_color"
     tools:context=".maps.mapsforge.v6.NewMap" >
 
     <!-- cache name and icon -->

--- a/main/res/values/themes.xml
+++ b/main/res/values/themes.xml
@@ -215,6 +215,7 @@
 
     <style name="Dialog_Alert" parent="@style/Theme.AppCompat.Dialog.Alert">
         <item name="text_color">@color/text_dark</item>
+        <item name="text_color_grey">@color/text_grey_dark</item>
         <item name="buttonBarButtonStyle">@style/button_borderless</item>
         <item name="background">?android:attr/selectableItemBackground</item>
         <item name="colorAccent">@color/colorAccent</item>
@@ -224,6 +225,7 @@
 
     <style name="Dialog_Alert_light" parent="@style/Theme.AppCompat.Light.Dialog.Alert">
         <item name="text_color">@color/text_light</item>
+        <item name="text_color_grey">@color/text_grey_light</item>
         <item name="buttonBarButtonStyle">@style/button_borderless</item>
         <item name="background">?android:attr/selectableItemBackground</item>
         <item name="colorAccent">@color/colorAccent</item>

--- a/main/src/cgeo/geocaching/activity/ActivityMixin.java
+++ b/main/src/cgeo/geocaching/activity/ActivityMixin.java
@@ -55,10 +55,7 @@ public final class ActivityMixin {
     }
 
     private static int getThemeId() {
-        if (Settings.isLightSkin()) {
-            return R.style.light;
-        }
-        return R.style.dark;
+        return Settings.isLightSkin() ? R.style.light : R.style.dark;
     }
 
     public static void setTheme(final Activity activity) {
@@ -70,11 +67,7 @@ public final class ActivityMixin {
     }
 
     public static int getDialogTheme() {
-        // Light theme dialogs don't work on Android Api < 11
-        if (Settings.isLightSkin()) {
-            return R.style.popup_light;
-        }
-        return R.style.popup_dark;
+        return Settings.isLightSkin() ? R.style.popup_light : R.style.popup_dark;
     }
 
     /**

--- a/main/src/cgeo/geocaching/apps/navi/GoogleNavigationApp.java
+++ b/main/src/cgeo/geocaching/apps/navi/GoogleNavigationApp.java
@@ -87,8 +87,9 @@ abstract class GoogleNavigationApp extends AbstractPointNavigationApp {
          * show a selection of all parking places and the cache itself, when using the navigation for driving
          */
         private void selectDriveTarget(final Context context, final ArrayList<IWaypoint> targets) {
-            final LayoutInflater inflater = LayoutInflater.from(context);
-            final ListAdapter adapter = new ArrayAdapter<IWaypoint>(context, R.layout.cacheslist_item_select, targets) {
+            final Context themeContext = Dialogs.newContextThemeWrapper(context);
+            final LayoutInflater inflater = LayoutInflater.from(themeContext);
+            final ListAdapter adapter = new ArrayAdapter<IWaypoint>(themeContext, R.layout.cacheslist_item_select, targets) {
                 @Override
                 public View getView(final int position, final View convertView, @NonNull final ViewGroup parent) {
 

--- a/main/src/cgeo/geocaching/ui/dialog/Dialogs.java
+++ b/main/src/cgeo/geocaching/ui/dialog/Dialogs.java
@@ -847,11 +847,11 @@ public final class Dialogs {
         builder.create().show();
     }
 
-    public static AlertDialog.Builder newBuilder(final Activity activity) {
-        return new AlertDialog.Builder(new ContextThemeWrapper(activity, Settings.isLightSkin() ? R.style.Dialog_Alert_light : R.style.Dialog_Alert));
+    public static AlertDialog.Builder newBuilder(final Context context) {
+        return new AlertDialog.Builder(newContextThemeWrapper(context));
     }
 
-    public static AlertDialog.Builder newBuilder(final Context context) {
-        return new AlertDialog.Builder(new ContextThemeWrapper(context, Settings.isLightSkin() ? R.style.Dialog_Alert_light : R.style.Dialog_Alert));
+    public static ContextThemeWrapper newContextThemeWrapper(final Context context) {
+        return new ContextThemeWrapper(context, Settings.isLightSkin() ? R.style.Dialog_Alert_light : R.style.Dialog_Alert);
     }
 }

--- a/main/src/cgeo/geocaching/utils/BackupUtils.java
+++ b/main/src/cgeo/geocaching/utils/BackupUtils.java
@@ -180,8 +180,7 @@ public class BackupUtils {
         }
 
         // We are using ContextThemeWrapper to prevent crashes caused by missing attribute definitions when starting the dialog from MainActivity
-        final Context c = new ContextThemeWrapper(activityContext, Settings.isLightSkin() ? R.style.Dialog_Alert_light : R.style.Dialog_Alert);
-        final View content = LayoutInflater.from(c).inflate(R.layout.restore_dialog, null);
+        final View content = LayoutInflater.from(Dialogs.newContextThemeWrapper(activityContext)).inflate(R.layout.restore_dialog, null);
         final CheckBox databaseCheckbox = content.findViewById(R.id.database_check_box);
         final CheckBox settingsCheckbox = content.findViewById(R.id.settings_check_box);
         final TextView warningText = content.findViewById(R.id.warning);

